code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -670,7 +686,11 @@
 	nonlethal_projectile_sound = 'sound/weapons/taser2.ogg'
 	lethal_projectile = /obj/item/projectile/beam/laser
 	lethal_projectile_sound = 'sound/weapons/laser.ogg'
-	desc = "An energy blaster auto-turret."
+	desc = "An energy blaster auto-turret. Use a welder to fix."
+	explosion_block = 50
```

Comment:
Change this with armor values:
```
	armor = list("melee" = 50, "bullet" = 30, "laser" = 30, "energy" = 30, "bomb" = 50, "bio" = 0, "rad" = 0, "fire" = 90, "acid" = 90)
```
That's the base turret armor plus 20 extra points of bomb armor.